src/main/java/core/basesyntax/garage/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.garage;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly, the stopWork method should print a more descriptive message that includes the specific type of 'Machine' that has stopped working.